File: app/src/main/java/mb00/android/codehub/api/model/Comment.java
package mb00.android.codehub.api.model;

import com.google.gson.annotations.SerializedName;


public class Comment {

    @SerializedName("user")
    private User user;
    @SerializedName("created_at")
    public String creationDate;
    @SerializedName("body")
    public String body;

    public User getUser() {
        return user;
    }

    public String getCreationDate() {
        return creationDate;
    }

    public String getBody() {
        return body;
    }

}

File: app/src/main/java/mb00/android/codehub/api/model/Comment.kt
package mb00.android.codehub.api.model

import com.google.gson.annotations.SerializedName


data class Comment(
        @SerializedName("user") var user: User?,
        @SerializedName("created_at") var creationDate: String?,
        @SerializedName("body") var body: String?
)
